force-app/main/default/classes/ServiceSessionSelector_TEST.cls

```diff
@@ -0,0 +1,137 @@
+        serviceSession1.PrimaryServiceProvider__c = newContact.Id;
+        serviceSession1.SessionStart__c = System.Today().addDays(-2);
+        serviceSessions.add(serviceSession1);
+
+        ServiceSession__c serviceSession2 = new ServiceSession__c();
+        serviceSession2.Name = 'Session 2';
+        serviceSession2.Status__c = 'Pending';
+        serviceSession2.PrimaryServiceProvider__c = newContact.Id;
+        serviceSession2.SessionStart__c = System.Today();
+        serviceSessions.add(serviceSession2);
+
+        ServiceSession__c serviceSession3 = new ServiceSession__c();
+        serviceSession3.Name = 'Session 3';
+        serviceSession3.Status__c = 'Pending';
+        serviceSession3.PrimaryServiceProvider__c = newContact.Id;
+        serviceSession3.SessionStart__c = System.Today().addDays(-7);
+        serviceSessions.add(serviceSession3);
+
+        insert serviceSessions;
+    }
+
+    @IsTest
+    private static void testGetServiceSessionsForThisWeek() {
+        final String dateRange = 'THIS_WEEK';
+
+        List<ServiceSession__c> expectedSessions = [
+            SELECT Id, Name, PrimaryServiceProvider__c, Status__c, SessionStart__c
+            FROM ServiceSession__c
+            WHERE SessionStart__c = THIS_WEEK
+            ORDER BY SessionStart__c
+        ];
+
+        Test.startTest();
+        ServiceSessionSelector sessionSelector = new ServiceSessionSelector();
+        List<ServiceSession__c> actualSessions = sessionSelector.getServiceSessions(
+            dateRange
+        );
+        Test.stopTest();
+
+        System.assertEquals(
+            expectedSessions.size(),
+            actualSessions.size(),
+            'The list size of actual Sessions is not the same as the list size of expected Sessions'
+        );
+    }
+
+    @IsTest
+    private static void testGetServiceSessionsForLastWeek() {
+        final String dateRange = 'LAST_WEEK';
+
+        List<ServiceSession__c> expectedSessions = [
+            SELECT Id, Name, PrimaryServiceProvider__c, Status__c, SessionStart__c
+            FROM ServiceSession__c
+            WHERE SessionStart__c = LAST_WEEK
+            ORDER BY SessionStart__c
+        ];
+
+        Test.startTest();
+        ServiceSessionSelector sessionSelector = new ServiceSessionSelector();
+        List<ServiceSession__c> actualSessions = sessionSelector.getServiceSessions(
+            dateRange
+        );
+        Test.stopTest();
+
+        System.assertEquals(
+            expectedSessions.size(),
+            actualSessions.size(),
+            'The list size of actual Sessions is not the same as the list size of expected Sessions'
+        );
+    }
+
+    @IsTest
+    private static void testGetServiceSessionsNoAccess() {
+        final String dateRange = 'LAST_WEEK';
+
+        Profile p = [SELECT Id FROM Profile WHERE Name = 'Standard User'];
+        Integer random = Integer.valueOf(math.rint(math.random() * 1000000));
```

Comment:
I believe we have a method in the test data factory or the test util for creating this standard user.